### DISCUSSION TITLE
Landing page using pure react js without typescript

### DISCUSSION THREAD

**Discussion by throne1986** [prevwong/craft.js#283] [General]
Hi everyone,

I want to run the landing example using pure react js without typescript, so I have refactored the given example in this repo, 

Here is my new repo : [https://github.com/throne1986/landing-app](url)

But, unfortunately, I keep getting the error:  `TypeError: r is not a function

![image](https://user-images.githubusercontent.com/27904724/127243905-c11b3d98-1360-4273-890e-ca68752ea058.png)

I am out of idea what I am doing wrong, can someone please tell me what is wrong here?

Thanks

**Comment by ankri** [comment]
related -> #285
